scenes/game/game.tscn

```diff
@@ -29,6 +29,8 @@
 
 [node name="GameScene" type="Node2D"]
 script = ExtResource("1_4f5jr")
+metadata/_edit_vertical_guides_ = [8.0, 208.0, 1056.0, 1161.0]
```

Comment:
What is this? Did you move some doodads in the editor?
Just delete these lines.

Reply:
this is godot storing in-editor guides for aligning ui elements (I fixed a bunch of misalignments)